app/facades/foods_index_search_total_facade.rb

```diff
@@ -0,0 +1,8 @@
+class FoodsIndexSearchTotalFacade
+
+    def self.service(search) 
+        data = FoodService.food_search(search)
+       
+        @total_foods = data[:totalHits]
```

Comment:
Consider trying to minimize the # of API calls you are making. Without caching (which we will learn about in week 5), we're making 2 of the same calls here. But you could refactor this to grab the total hits data in the first call (keep in mind that Ruby allows you to return more than one object from a method)